### PR TITLE
NTR: paraspinal and pericardial regions

### LINKED ISSUE #2895: NTR: paraspinal and pericardial regions
**Preferred term label:** paraspinal region
**Synonym** EXACT: paraspinal [NCIT:C129461]
**Definition:** The region of the body adjacent to the spinal column. [NCIT:C166000]
**Parent term**: dorsal region element
**ORCID**: 0000-0002-0819-0473

**Preferred term label:** pericardial region
**Definition**: The region of the body adjacent to the pericardium and heart. (0000-0002-0819-0473)
**Parent term**: trunk region element

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2895

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224018,6 +224018,23 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+synonym: "paraspinal" EXACT [NCIT:C129461]
+is_a: UBERON:0005174 ! dorsal region element
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date "2023-05-16T06:37:51Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480061
+name: pericardial region
+def: "The region of the body adjacent to the pericardium and heart." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0005177 ! trunk region element
```
Comment (gouttegd) [inline src/ontology/uberon-edit.obo:224034]: Same concern as with the previous term. [Trunk region element](http://purl.obolibrary.org/obo/UBERON_0005177) is defined as an [organ](http://purl.obolibrary.org/obo/UBERON_0000062). I’d suggest [subdivision of trunk](http://purl.obolibrary.org/obo/UBERON_0009569) as a parent.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224018,6 +224018,23 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 property_value: curator_notes "The perigastric lymph nodes include at least the right paracardial lymph nodes, left paracardial lymph nodes, lesser curvature lymph nodes, greater curvature lymph nodes, suprapyloric lymph nodes and infrapyloric lymph nodes. PMID:26973384" xsd:string
 property_value: dcterms-date "2023-05-11T12:28:12Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480060
+name: paraspinal region
+def: "The region of the body adjacent to the vertebral column." [NCIT:C166000]
```
Comment (gouttegd) [inline src/ontology/uberon-edit.obo:224024]: Given that definition, maybe consider adding an explicit taxon restriction to vertebrates?

**Review by gouttegd — CHANGES_REQUESTED** [review]

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224018,6 +224018,23 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+synonym: "paraspinal" EXACT [NCIT:C129461]
+is_a: UBERON:0005174 ! dorsal region element
```
[the remark this answers is not in the document]
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224026]: @gouttegd I made it child of "thoracic region of vertebral column". But now I think is better child of "posterior region of body". What do you think?

[the remark this answers is not in the document]
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224026]: @gouttegd or 'subdivision of trunk'?

[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/uberon-edit.obo:224026]: _Thoracic region of vertebral column_ doesn’t sound right, for two reasons:

* it limits the region to the thorax, while the vertebral column extends beyond it;
* that would make the paraspinal region a part of the vertebral column itself, while it is supposed to the region adjacent to it.

_Subdivision of trunk_ could be good, with the caveat that [trunk](http://purl.obolibrary.org/obo/UBERON_0002100) is defined as “the part of the body _posterior to the cervical region_” – so again, using that as a parent would limit the extent of the paraspinal region to the region of the body adjacent to the thoraco-abdominal segment of the column, excluding the cervical segment. Now if that term is commonly used with that meaning, it’s good, but then the textual definition should be amended to reflect that.

[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/uberon-edit.obo:224026]: _posterior region of body_ looks good.

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -2164,7 +2164,7 @@ xref: SCTID:118966000
 xref: UMLS:C1519343 {source="ncithesaurus:Skeletal_System_Part"}
 is_a: UBERON:0011216 {source="cjm"} ! organ system subdivision
 relationship: composed_primarily_of UBERON:0004765 ! skeletal element
-relationship: only_in_taxon NCBITaxon:7711 ! Chordata
+relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
```
Comment (gouttegd) [inline src/ontology/uberon-edit.obo:2167]: When I suggested adding an explicit taxon restriction to vertebrates, I meant adding it to the new _paraspinal region_ term. No reason to change the existing taxon restriction on [subvidision of skeletal system](http://purl.obolibrary.org/obo/UBERON_0000075).

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:2167]: @gouttegd yes, sorry, I will correct it now

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -3341,7 +3341,7 @@ relationship: only_in_taxon NCBITaxon:33213 ! Bilateria
 id: UBERON:0000154
 name: posterior region of body
 is_a: UBERON:0000475 ! organism subdivision
-relationship: only_in_taxon NCBITaxon:33213 ! Bilateria
+relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
```
Comment (gouttegd) [inline src/ontology/uberon-edit.obo:3344]: Again: please add the taxon restriction directly to the new term _paraspinal region_ that you are adding in this PR, not to any parental term.

**Review by gouttegd — APPROVED** [review]
Looks all good now. When the CI check is complete, you can merge. :)

**Pull request merged**